addons/payment_razorpay/views/payment_form_templates.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="payment_razorpay.token_form" inherit_id="payment.token_form">
+        <xpath expr="//t[@t-if='is_test']" position="before">
+            <t t-set="razorpay_limit_exceed_warning" t-value="token_sudo._get_razorpay_limit_exceed_warning(amount)"/>
+        </xpath>
+        <xpath expr="//t[@t-if='is_test']" position="inside">
+            <t t-set="title" t-if="razorpay_limit_exceed_warning and not is_test" t-value="razorpay_limit_exceed_warning"/>
```

Comment:
```suggestion
        <xpath expr="//t[@t-if='is_test']" position="inside">
            <t t-set="title" t-if="razorpay_limit_exceed_warning and not is_test" t-value="razorpay_limit_exceed_warning"/>
```
This is inside the `t-if='is_test'`, so this `t-set t-if` will never meet the condition `not is_test`, which then means that this warning will only be shown in test mode, and never in production :thinking:

Reply:
No, Actually I did changed the condition :sweat_smile: But I was below this code :sweat:

Reply:
Ha yes, indeed, didn't realise that it'd cancel that comment when I saw it, sorry for that '^^